### ISSUE TITLE
easily change font size

### ISSUE THREAD

**Issue by sanette** [xournalpp/xournalpp#3019]
With Xournal when you entered some text it was easy to change the font size on the fly. I don't see how to do this in Xournal++.
Maybe I missed something, but the only thing I found is to call Tools/Text font, but then I can't simply change size, I have to select a font. I don't want to select a font because I don't even know which font was selected to start with. (And by the way I don't know either which size was selected, because the "size" slider is on "1" by default)

**Comment by rolandlo** [comment]
You don't write what version you use. Can you find the Font Button? E.g. in the Xournal++ toolbar it is right of the colors:
![grafik](https://user-images.githubusercontent.com/40485433/114427914-ddf5df80-9bbb-11eb-941f-0db0a842337b.png)
Just click on it and you can change the font size.
In version 1.1.0+dev the font size is updated according to the text that is currently edited. Usually a font ("Sans" and font size (12pt) is selected by default. It is saved in the `settings.xml` file.

**Comment by sanette** [comment]
yes, I'm using 1.1.0+dev, but I don't see the font button, here is what I get
![Screenshot_20210412_225008](https://user-images.githubusercontent.com/7536536/114460178-bade2680-9be1-11eb-8b5b-a490cfe070e0.png)

**Comment by rolandlo** [comment]
Seems your screen resolution is too small, so there is not enough space for the font button. With the `Portrait` toolbar (selectable under menu `View > Toolbars` you should see it.

**Comment by sanette** [comment]
Thanks for the answer. Well, my screen resolution is 3200x1800, I hope that's not considered small ;)
I guess because of my DE scale factor, it amounts to a "small" resolution. (Maybe I'm not the only user affected by this.
A solution could be to display the font dialog (in a prominent place) only when the text toolis active.)
 
If I select the `Portrait` toolbar, I do see the dialog, but then I'm hit by the second problem I mentioned earlier: the default size is 0, and there is no default font. Hence I cannot just change the size (it forces me to select a font too)
![Screenshot_20210413_094645](https://user-images.githubusercontent.com/7536536/114516512-c3fce100-9c3d-11eb-8038-36816d2e52ef.png)

**Comment by Photon89** [comment]
> the default size is 0, and there is no default font. Hence I cannot just change the size (it forces me to select a font too)

Afaik this is fixed in the 1.1.0 dev version (which is not yet released, compile the git tree to get it).

**Comment by rolandlo** [comment]
You don't need to compile the git tree, you can download it from [the nightly releases](https://github.com/xournalpp/xournalpp/releases/tag/nightly)

If your system doesn't have the default font "Sans", you can create a `xopp`-file with some text, `gunzip` it to see what font was used and make that the new default.

**Comment by sanette** [comment]
> Afaik this is fixed in the 1.1.0 dev version (which is not yet released, compile the git tree to get it).

indeed! thanks, I should have checked before. So for me the only point remaining would be to find a way to show the font/size without having to use the Portrait view; but that's not so crucial because I can always use CTRL-SHIFT-F
